akka-stream/src/main/scala/akka/stream/scaladsl/FlowWithContext.scala

```diff
@@ -101,15 +210,10 @@ final class FlowWithContext[-CtxIn, -In, +CtxOut, +Out, +Mat](
   override def withAttributes(attr: Attributes): Repr[CtxOut, Out] = new FlowWithContext(underlying, traversalBuilder.setAttributes(attr), shape)
 
   override type Repr[+C, +O] = FlowWithContext[CtxIn @uncheckedVariance, In @uncheckedVariance, C, O, Mat @uncheckedVariance]
-  override type Prov[+C, +O] = Flow[(In @uncheckedVariance, CtxIn @uncheckedVariance), (O, C), Mat @uncheckedVariance]
 
   override def via[Ctx2, Out2, Mat2](viaFlow: Graph[FlowShape[(Out, CtxOut), (Out2, Ctx2)], Mat2]): Repr[Ctx2, Out2] = from(underlying.via(viaFlow))
 
-  def to[Mat2](sink: Graph[SinkShape[(Out, CtxOut)], Mat2]): Sink[(In, CtxIn), Mat] = underlying.toMat(sink)(Keep.left)
```

Comment:
> because you can do `asFlow.to(sink)`

Yes and because we want to keep the interfaces minimal. These ones can be useful but might not make the threshold.